packages/frontend/src/components/common/Filters/FilterInputs/DefaultFilterInputs.tsx

```diff
@@ -40,7 +40,9 @@ const DefaultFilterInputs = <T extends ConditionalRule>({
     const placeholder = getPlaceholderByFilterTypeAndOperator({
         type: filterType,
         operator: rule.operator,
-        disabled: isFilterRule(rule) ? rule.disabled : undefined,
+        disabled: isFilterRule(rule)
+            ? rule.disabled && !rule.values
```

Comment:
will `values` always be falsy in this case? what about `values` being set to `[]` 🤔

Reply:
Interestingly ![] is falsey. I can make this more explicit if you want.